defender/staged/downloaded/Forta_Liquidation_Monitor.spec.js

```diff
@@ -101,16 +110,16 @@ function createFortaSentinelEvent(finding, blockHash, txHash) {
 
 describe('check autotask', () => {
   it('Runs autotask against mock data and posts in Discord (manual-check)', async () => {
-    const autotaskEvent = createFortaSentinelEvent(mockFinding, mockBlockHash, mockTxHash);
+    const autotaskEvent = createFortaSentinelEvent(mockFinding, mockBlockHash);
```

Comment:
Use the approach created in AE-1313 to mock `axios` in order to capture any outgoing requests and then perform `expect()` against them.

Reply:
Addressed in commit 46775eb880c99b344c54edca625edb57827580e3